.github/workflows/superset-docs-deploy.yml

```diff
@@ -41,19 +40,28 @@ jobs:
           node-version: '18'
       - name: Setup Python
         uses: ./.github/actions/setup-backend/
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'zulu'
+          java-version: '21'
+      - name: Install Graphviz
```

Comment:
Note that this is not an Apache-compatible license, but for this use case, I _think_ it's Ok.